File: 프로그래머스/unrated/181879. 길이에 따른 연산/길이에 따른 연산.java
class Solution {
    public int solution(int[] num_list) {
        int answer = num_list.length < 11 ? 1 : 0;
        
        for(int i=0; i < num_list.length; i++){
            if(num_list.length < 11){
                answer *= num_list[i];       
            }
            else{
                  answer += num_list[i];  
            }
            
        }
        return answer;
    }
}

File: 프로그래머스/unrated/181879. 길이에 따른 연산/길이에 따른 연산.js
function solution(num_list) {
    var answer = 0;
    var sum = 0;
    var mul = 1;
   
    for(var i=0; i < num_list.length; i++){
        if(num_list.length >= 11 ){
        sum+=num_list[i]
        answer=sum;
        }
        else{
        mul*=num_list[i]
        answer=mul;     
        }
       
        
    }
    return answer;
}